pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -73,6 +73,9 @@ type NodeConfig struct {
 	// DNSConfig controls the DNS configuration.
 	DNSServer *dns.Server
 
+	// TODO: Make this configurable as currently it is not set which means no off-cluster bridging will be performed.
```

Comment:
switched to use c.ProxyConfig.ClusterCIDR, which is already configurable via `proxyArguments`